### PR TITLE
Add AI 101 to /ai

### PR DESCRIPTION
**Pull request by bethanyaconnor**
Adds a tile for AI 101 on code.org/ai

Two things I'd like feedback on:
- is this order correct?
- now that there are 3 offerings, the button text is squished. Should that be updated or is it okay as-is?

Screenshot 2024-05-13 at 9 07 57 AM

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/images/self-paced-pl-tile-ai-101.jpg
Comment (bethanyaconnor) [inline pegasus/sites.v3/code.org/public/images/self-paced-pl-tile-ai-101.jpg:1]: This is in the same location as it is in https://github.com/code-dot-org/code-dot-org/pull/58564, which should make a merge conflict easy to resolve.

**Review by kelbyhawn — COMMENTED** [review]
This looks good to me code-wise, but great questions 🤔 We could highlight one as a full-width block and have the other two underneath, similar to https://code.org/minecraft?

Let's post this in Slack and ask Dani and Moshe.

**Comment by dmcavoy** [comment]
@bethanyaconnor we can just change the button text to "Start module"

**Comment by bethanyaconnor** [comment]
@dmcavoy that looks better to me!

Screenshot 2024-05-14 at 9 43 31 AM

**Review by kelbyhawn — APPROVED** [review]

**Comment by bethanyaconnor** [comment]
published state is [stable](https://github.com/code-dot-org/code-dot-org/blob/8addf154365d8b216c46ab23ff0e79ccee143610/dashboard/config/courses/self-paced-pl-ai-101-2024.course#L10C23-L10C29) -- merging!

**Pull request merged**